src/main/java/chacha/Storage.java

```diff
@@ -0,0 +1,102 @@
+package chacha;
+
+import java.io.BufferedWriter;
```

Comment:
Maybe you should consider adding spaces in between import statements from different packages to improve readability.